README.md

```diff
@@ -0,0 +1,13 @@
+# UnzerGui Module
+[![CI](https://github.com/spryker-eco/unzer-gui/actions/workflows/ci.yml/badge.svg)](https://github.com/spryker-eco/unzer-gui/actions/workflows/ci.yml)
+[![Latest Stable Version](https://poser.pugx.org/spryker-eco/unzer-gui/v/stable.svg)](https://packagist.org/packages/spryker-eco/unzer-gui)
+[![License](https://img.shields.io/github/license/spryker-eco/unzer-gui.svg?b=master)](https://github.com/spryker-eco/unzer-gui)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/badges/build.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/unzer-gui/build-status/master)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg)](https://php.net/)
+## Installation
```

Comment:
Please, add small description of the module here.

Reply:
done